packages/react-cosmos-apollo-proxy/src/index.js

```diff
@@ -1,43 +1,50 @@
 import React, { Component } from 'react';
-import { makeExecutableSchema, addMockFunctionsToSchema } from 'graphql-tools';
-import { graphql, print } from 'graphql';
-import { ApolloProvider, ApolloClient } from 'react-apollo';
+import { ApolloClient } from 'apollo-client';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { HttpLink } from 'apollo-link-http';
+import { ApolloProvider } from 'react-apollo';
 import { proxyPropTypes } from 'react-cosmos-shared/lib/react';
+import { createFixtureLink } from './fixtureLink';
 
 const defaults = {
-  // Must provide schema definition with query type or a type named Query.
-  typeDefs: `type Query { hello: String }`,
-  context: {},
-  rootValue: {}
+  fixtureKey: 'apollo'
 };
 
 export default function createApolloProxy(options) {
-  const { typeDefs, mocks, context, rootValue } = { ...defaults, ...options };
-
-  const schema = makeExecutableSchema({ typeDefs });
-
-  if (mocks) {
-    addMockFunctionsToSchema({ schema, mocks });
-  }
+  const { fixtureKey, endpoint, client } = {
+    ...defaults,
+    ...options
+  };
 
   class ApolloProxy extends Component {
     constructor(props) {
       super(props);
 
-      this.client = new ApolloClient({
-        networkInterface: {
-          query(request) {
-            return graphql(
-              schema,
-              print(request.query),
-              rootValue,
-              context,
-              request.variables,
-              request.operationName
-            );
-          }
-        }
-      });
+      if (!endpoint && !client) {
+        throw new Error(
+          `
+It looks the Apollo Proxy is not configured! 
+Give it:
+- a GraphQL endpoint to send GraphQL operations to;
+- a configured Apollo Client (maybe the one you use in your app?);
+Read more at: https://github.com/react-cosmos/react-cosmos#react-apollo-graphql.`
+        );
+      }
+
+      const { resolveWith, failWith } = this.props.fixture[fixtureKey] || {};
+
+      this.client =
+        client ||
+        new ApolloClient({
+          cache: new InMemoryCache(),
```

Comment:
Something I ran into recently in my "hand rolled" version of this.  You probably are going to need to paramaterize dataIdFromObject at some point.